### PR TITLE
Tests for user profile and restore password functionality

### PR DESCRIPTION
**Pull request by TetianaFomina**
Added two spec files to test user profile (view and update) and restore password.

### CODE REVIEW

File: src/app/[language]/password-change/page-content.tsx [hunk 1]
```diff
@@ -118,13 +119,15 @@ function Form() {
...
                 type="password"
+                testId="password"
               />
             </Grid>
             <Grid item xs={12}>
               <FormTextInput<PasswordChangeFormData>
                 name="passwordConfirmation"
                 label={t("password-change:inputs.passwordConfirmation.label")}
                 type="password"
+                testId="passwordConfirmation"
```
Comment (Shchepotin) [inline src/app/[language]/password-change/page-content.tsx:130]: Let's use in `textId` [kebab case ](https://www.theserverside.com/definition/Kebab-case)

like:

```
testId="password-confirmation"
```

Please also check all files :)

File: cypress/fixtures/profileImage.jpg
Comment (Shchepotin) [inline cypress/fixtures/profileImage.jpg:1]: LOOK AT THIS DUDE :D

Comment (Shchepotin) [reply cypress/fixtures/profileImage.jpg:1]: Just resolve this comment after reading :D

File: cypress/e2e/1-auth/forgot-password.cy.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+      cy.getBySel("password").type(newPassword);
+      cy.getBySel("passwordConfirmation").type(newPassword);
+      cy.getBySel("set-password").click();
+      cy.wait("@resetPassword").then((request) => {
+        expect(request.response?.statusCode).to.equal(204);
+        cy.get("#notistack-snackbar").should("be.visible");
+      });
+      cy.location("pathname").should("include", "/sign-in");
+
+      //check user cannot login with old password
```
Comment (Shchepotin) [inline cypress/e2e/1-auth/forgot-password.cy.ts:129]:
```suggestion
      // check user cannot login with old password
```

File: cypress/e2e/1-auth/forgot-password.cy.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      cy.getBySel("email").type(email);
+      cy.getBySel("password").type(password);
+      cy.getBySel("sign-in-submit").click();
+      cy.wait("@login").then((request) => {
+        expect(request.response?.statusCode).to.equal(422);
+        cy.getBySel("password-error").should("be.visible");
+      });
+
+      //check user is allowed to login with the new password
```
Comment (Shchepotin) [inline cypress/e2e/1-auth/forgot-password.cy.ts:138]:
```suggestion
      // check user is allowed to login with the new password
```

**Review by Shchepotin — CHANGES_REQUESTED** [review]

File: src/app/[language]/password-change/page-content.tsx (hunk 1 above)
Comment (TetianaFomina) [reply src/app/[language]/password-change/page-content.tsx]: Done

File: cypress/e2e/1-auth/user-profile.cy.ts [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+    });
+    cy.getBySel("save-profile").click();
+    cy.wait("@profileUpdate").then((request) => {
+      expect(request.response?.statusCode).to.equal(200);
+      cy.getBySel("cancel-edit-profile").click();
+      cy.get("img").should("be.visible").and("have.attr", "src");
+    });
+  });
+
+  it("Whant to leave page", () => {
```
Comment (Shchepotin) [inline cypress/e2e/1-auth/user-profile.cy.ts:95]:
```suggestion
  it("Want to leave page", () => {
```

File: src/app/[language]/profile/edit/page-content.tsx [hunk 5]
```diff
@@ -296,6 +304,7 @@ function FormChangePassword() {
                   color="inherit"
                   LinkComponent={Link}
                   href="/profile"
+                  data-testid="cancel-editPassword"
```
Comment (Shchepotin) [inline src/app/[language]/profile/edit/page-content.tsx:307]:
```suggestion
                  data-testid="cancel-edit-password"
```

**Review by Shchepotin — APPROVED** [review]

**Pull request merged**